tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx

```diff
@@ -110,6 +110,11 @@ void ROOT::Experimental::RNTupleDescriptor::PrintInfo(std::ostream &output) cons
       info.fElementSize = elementSize;
       info.fType = column.second.GetModel().GetType();
 
+      // Alias columns (columns of projected fields) don't contribute to the storage consumption. Count them
+      // but don't add the the page sizes to the overall volume.
+      if (column.second.IsAliasColumn())
+         continue;
+
```

Comment:
As-is, `info` is not pushed to the `columns` vector, and thus not displayed in the column summary in the `COLUMN DETAILS` section.
I guess that's intentional, but then maybe let's move this to line 101, to avoid doing unnecessary work :slightly_smiling_face:.